app/konnect/org-management/okta-idp.md

```diff
@@ -186,9 +219,41 @@ application into {{site.konnect_saas}}.
     * The path does not require a slash (`/`).
 
 1. Click **Save**.
+1. From the list of authentication providers, open the context menu and **Enable OIDC**.
+{% endnavtab %}
+{% navtab SAML %}
+1. In another separate browser tab, log in to [{{site.konnect_saas}}](https://cloud.konghq.com).
+1. Click {% konnect_icon organizations %} **Organization** > **Settings**, then **Authentication Schemes**.
+1. Click **Configure provider** for **SAML**.
+
+1. In Okta, locate your Metadata URL:
+    1. Go to **Sign On** page in the Okta application created in the previous step.
+    2. Copy the **IDP Metadata URL** under the Settings section. It should look like:
+
+             https://<your-okta-domain>.okta.com/app/exkgzjkl0kUZB06Ky5d7/sso/saml/metadata
+
+1. In the **Login Path** box, enter a unique string. For example: `examplepath`.
+
+   {{site.konnect_short_name}} uses this string to generate a custom login
+   URL for your organization.
+
+   Requirements:
+    * The path must be unique *across all {{site.konnect_short_name}} organizations*.
+      If your desired path is already taken, you must choose another one.
+    * The path can be any alphanumeric string.
+    * The path does not require a slash (`/`).
+
+
+1. Click **Save**.
+1. From the list of authentication providers, open the context menu and **Enable SAML**.
+1. In Okta update the placeholder **Single Sign-On URL** and **Audience URI (SP Entity ID)** that you set in the previous section.
```

Comment:
Let's not forget to mention the SAML provider needs to be enabled. Let's add another line along those lines?
```
1. Close the configuration dialog and click **Enable** on your SAML provider.
```